### ISSUE TITLE
Impossible to highlight active element when using TourMatMenuModule in Angular Material project

### ISSUE THREAD

**Issue by hakimio** [isaacplmann/ngx-tour#125]
Having a transparent backdrop and highlighting active element (raising it above the backdrop) seems to be impossible when using TourMatMenuModule inside Angular Material application. While the tour step stays above the backdrop, the active element does not go up.

Following is the CSS tested (yes, the CSS is applied correctly - checked in the dom inspector quite a few times):
```
.touranchor--is-active {
    position: relative;
    z-index: 10000;
   background: white;
}

.cdk-overlay-transparent-backdrop, .cdk-overlay-transparent-backdrop.cdk-overlay-backdrop-showing {
    opacity: 0.75;
    background: black;
}
```

PS: I know that it works in your simple demo app.
PPS: let me know if you need a test app.

**Comment by isaacplmann** [comment]
Yes, providing steps to reproduce would be helpful.  I see you've already looked at #97 which has a long discussion about this issue.

**Comment by hakimio** [comment]
The steps would be:
* Create new angular material app (you can use latest version of ng cli to generate it nowadays).
* Try to use the above CSS in your global style.css file.

And yes, I saw the old issue, but unfortunately the solution you proposed doesn't work.
Anyway, I'll try to create a demo project for you.

**Comment by hakimio** [comment]
I have created a test app for the issue described here (basically auto-generated Angular Material app + ngx-tour + the CSS you recommended to use):
https://github.com/hakimio/ngx-tour-bugs

If you can make it work, I'll take a look if I can add support for optional "anchorId" and "placement" config in TourMatMenuModule.

**Comment by hakimio** [comment]
@isaacplmann
Must be because of [stacking contexts](https://philipwalton.com/articles/what-no-one-told-you-about-z-index/). Anyway, how difficult it would be to implement html element cloning like they have in "angular-ui-tour"? Can you give me some pointers?

**Comment by hakimio** [comment]
Just found out another simpler solution: just create a cut out in the backdrop like [ngx-joyride](https://github.com/tnicola/ngx-joyride) did.

**Comment by isaacplmann** [comment]
That's very clever.  I'd love to merge a PR with that functionality.  I was really dreading all the bugs related to cloning content.

**Comment by hakimio** [comment]
@isaacplmann
Have you seen my [PR](https://github.com/isaacplmann/ngx-tour/pull/127)?

**Comment by isaacplmann** [comment]
Yes, I hope to have time to review it tomorrow.

**Comment by isaacplmann** [comment]
Fixed by #127 in `ngx-tour-md-menu@3.1.0`
